test/route.js

```diff
@@ -1,8 +1,6 @@
 var PushStateTree = require('../src/pushStateTree');
 
 describe('PushStateTree should', function() {
-  'use strict';
```

Comment:
good catch! In the `package.json` I'm configuring babel to add 'use strict' by default, so we don't need to add on each file/function.

```json
"babel": {
    "presets": [
      "es2015"
    ],
    "plugins": [
      "transform-runtime",
      [
        "transform-strict-mode",
        {
          "strict": true
        }
      ]
    ]
  }
```

I forgot to remove the 'use strict' from the other files, I will do it before we merge.

Thanks!